src/server.c

```diff
@@ -526,7 +528,7 @@ void server_init(bool test)
     tonies_init();
     if (get_settings()->core.tonies_json_auto_update || test)
     {
-        tonies_update();
```

Comment:
Please use the osCreateTask as shown here, so we are platform independent
https://github.com/toniebox-reverse-engineering/teddycloud/blob/21f9c08e3daa13dde35991f7f568674f2e98641f/src/cyclone/cyclone_tcp/http/http_server.c#L211